src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -3143,8 +3143,7 @@ SupportedLateralQuery(Query *parentQuery, Query *lateralQuery)
 	{
 		OpExpr *operatorExpression = NULL;
 		List *argumentList = NIL;
-		char *operatorName = NULL;
-		int equalsOperator = 0;
+		bool equalsOperator = false;
```

Comment:
Changing the type of a variable from int to bool without changing the name is a bit suspicious.